selenium-system-test/src/test/scala/htsstride/pages/eligibility/ErrorSCREligiblePage.scala

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
We make a page object per page, so these methods for error validation can be moved to SCRCustomerEligiblePage